src/scripts/main.js

```diff
@@ -2,7 +2,25 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tableBody = document.querySelector('tbody');
 
-// write your code here
+people.map(person => {
+  tableBody.insertAdjacentHTML('beforeend', `
+    <tr>
+      <th>${person.name}</th>
```

Comment:
We should use td tag for the table cells

Reply:
Thanks, already fixed.